src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinternetbankingpayment.phtml

```diff
@@ -1,59 +1,26 @@
 <?php
 $code = $this->getMethodCode();
+// These will eventually come from capabilities API
+$bankCodes = ['internet_banking_scb', 'internet_banking_ktb', 'internet_banking_bay', 'internet_banking_bbl'];
 ?>
 
 <ul id="payment_form_<?php echo $code; ?>" class="banks-list form-list" style="display:none;">
-    <!-- SCB -->
-    <li class="item">
-        <input class="validate-one-required-by-name" id="internet_banking_scb" type="radio" name="payment[bank_type]" value="internet_banking_scb" />
-        <label for="internet_banking_scb">
-            <div class="omise-logo-wrapper scb">
-                <div class="logo-scb"></div>
-            </div>
-            <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$this->__('Siam Commercial Bank');?></span><br/>
-            </div>
-        </label>
-    </li>
 
-    <!-- KTB -->
+    <?php foreach($bankCodes as $bankCode) {  $shortCode = str_replace('internet_banking_', '', $bankCode); ?>
```

Comment:
in HTML we should use construction:
```
<?php foreach ($bankCodes as $bankCode): ?>


<?php endforeach; ?>
```

for me code:
```
<?php if ($a) { ?>

<div>hello</div>

<?php } ?>
```

Is looking the same bad as this:

```
<?php
public function foo()
{
?>

<div>hello</div>

<?php 
}
?>